### DISCUSSION TITLE
Background platforming

### DISCUSSION THREAD

**Discussion by ShyGuy324853** [SuperTux/supertux#2857] [Ideas]
There could be a new block that you stand on, and jumping can transport you to the background, and playing levels in a background will be gamechanging.

![Screenshot 2023-10-20 7 16 27 PM](https://github.com/SuperTux/supertux/assets/148596557/015e3f2f-d89e-4b9f-b026-dd44679d28f8)

**Comment by tylerandari13** [comment]
Oh yeah like that one Wario game on the VirtualBoy.

Also why was this closed lmao.

**Comment by mrkubax10** [comment]
> Also why was this closed lmao.

Maybe misclick

**Comment by FunGame2021** [comment]
Like Super Mario Wonder, it would be spectacular.

**Comment by biggeryetbetter** [comment]
I think this is already possible with careful use of scripting: 
- Have a sector for each background layer that puts it in full detail while putting the others in lower detail
- Use script triggers and an intuitive design element such as a bridge to transport Tux from one layer to the other such that they remain topologically consistent (or not, non-euclidean Antarctica could be fun as well)

No additional coding on the part of the developers is required, maybe just a few sprites of a bridge.

**Comment by bruhmoent** [comment]
But how can you achieve the tux changing sprite and hitbox size with scripting? Haven't heard about it.

**Comment by ShyGuy324853** [comment]
It was my first time using GitHub, and I didn't know what to do.
